### DISCUSSION TITLE
The video quality

### DISCUSSION THREAD

**Discussion by CQZ66** [qnblackcat/uYouPlus#740] [Q&A]
The video quality is up to 1080p.I can't watch 4k video.Does anyone know how to use it?

**Comment by pha1n0q** [comment]
Use YTUHD - To enable this, open YouTube's settings > Video quality preferences > Enable VP9 decoding.

**Reply by softsoundd** [reply]
This toggle is definitely working on my 14PM, but can I get some clarification on what the text at the bottom means? ![image](https://user-images.githubusercontent.com/90562023/210236868-b858b415-4341-43d1-834b-4fec93143e76.png)

Shouldn't this equal "1" on supported devices?
